### PR TITLE
Add new ConstructionCone class to DetectionClass.

### PR DESCRIPTION
**Pull request by dersim-davaod**
Checks:

* [x] Update changelog
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:

### CODE REVIEW

File: MapboxVision/View/DetectionsView.swift [hunk 1]
```diff
@@ -62,6 +62,8 @@ private extension DetectionClass {
             return UIColor(red: 239.0 / 255.0, green: 6.0 / 255.0, blue: 255.0 / 255.0, alpha: 1.0)
         case .bicycle:
             return UIColor(red: 0, green: 165.0 / 255.0, blue: 1.0, alpha: 1.0)
+        case .constructionCone:
+            return UIColor(red: 255.0 / 255.0 , green: 255.0 / 255.0, blue: 99.0 / 255.0, alpha: 1.0)
```
Comment (chezzdev) [inline MapboxVision/View/DetectionsView.swift]:
```suggestion
            return UIColor(red: 255.0 / 255.0, green: 255.0 / 255.0, blue: 99.0 / 255.0, alpha: 1.0)
```

File: MapboxVision/View/DetectionsView.swift [hunk 2]
```diff
@@ -77,6 +79,8 @@ private extension DetectionClass {
             return "Person"
         case .bicycle:
             return "Bicycle"
+        case .constructionCone:
+            return "Construction Cone"
```
Comment (chezzdev) [inline MapboxVision/View/DetectionsView.swift]: I'd go for something shorter not to overcrowd the screen.
```suggestion
            return "Cone"
```

Comment (dersim-davaod) [reply MapboxVision/View/DetectionsView.swift]: Agree.

File: MapboxVision/View/DetectionsView.swift (hunk 1 above)
Comment (dersim-davaod) [reply MapboxVision/View/DetectionsView.swift]: Updated.

**Pull request merged**